src/Seksjon/Sprak.elm

```diff
@@ -0,0 +1,718 @@
+
+        ErrorLogget result ->
+            ( Model model, Cmd.none ) |> IkkeFerdig
+
+        SpråkkoderHentet result ->
+            case result of
+                Ok koder ->
+                    ( Model { model | språkKoder = Success koder }
+                    , SamtaleAnimasjon.scrollTilBunn ViewportSatt
+                    )
+                        |> IkkeFerdig
+
+                Err error ->
+                    ( nesteSamtaleStegUtenMelding model (SpråkkodeneFeilet error)
+                    , Cmd.batch
+                        [ logFeilmelding error "Språkkoder Hentet"
+                        , SamtaleAnimasjon.scrollTilBunn ViewportSatt
+                        ]
+                    )
+                        |> IkkeFerdig
+
+        ValgtSpråk valgtSpråk ->
+            case model.språkKoder of
+                Loading cmd ->
+                    ( Model model, cmd ) |> IkkeFerdig
+
+                Success list ->
+                    ( Model { model | aktivSamtale = VelgNyttSpråk (List.find (\sprakkode -> valgtSpråk == Sprakkoder.kode sprakkode) list) }
+                    , Cmd.none
+                    )
+                        |> IkkeFerdig
+
+                Failure error ->
+                    ( Model model, Cmd.none ) |> IkkeFerdig
+
+        BrukerHarValgtSpråk ->
+            case model.aktivSamtale of
+                VelgNyttSpråk språkkode ->
+                    case språkkode of
+                        Just kode ->
+                            ( LeggTilFerdighetMuntlig (Sprakkoder.kode kode)
+                                |> nesteSamtaleSteg model (Melding.svar [ Sprakkoder.kode kode ])
+                            , SamtaleAnimasjon.scrollTilBunn ViewportSatt
+                            )
+                                |> IkkeFerdig
+
+                        Nothing ->
+                            ( Model model, Cmd.none ) |> IkkeFerdig
+
+                _ ->
+                    ( Model model, Cmd.none ) |> IkkeFerdig
+
+        SendSkjemaPåNytt språkSkjema nyModel ->
+            ( nyModel, leggTilSpråkAPI språkSkjema ) |> IkkeFerdig
+
+        BrukerVilHenteSpråkPåNytt ->
+            ( Model model, hentSpråkkoder ) |> IkkeFerdig
+
+        StartÅSkrive ->
+            ( Model
+                { model
+                    | seksjonsMeldingsLogg =
+                        MeldingsLogg.startÅSkrive model.seksjonsMeldingsLogg
+                }
+            , Cmd.batch
+                [ SamtaleAnimasjon.scrollTilBunn ViewportSatt
+                , Process.sleep 1000
+                    |> Task.perform (\_ -> FullførMelding)
+                ]
+            )
+                |> IkkeFerdig
+
+        FullførMelding ->
+            model.seksjonsMeldingsLogg
+                |> MeldingsLogg.fullførMelding
+                |> updateEtterFullførtMelding model
+
+
+updateEtterFullførtMelding : ModelInfo -> MeldingsLogg -> SamtaleStatus
+updateEtterFullførtMelding model nyMeldingsLogg =
+    {--
```

Comment:
Kommentert ut kode

Reply:
Jeg tror vi kommer til å få bruk for den når vi finner ut hvordan den funker